doc/tech/README.md

```diff
@@ -3,203 +3,166 @@
+* [Reference Architecture Principles](../ref_arch#principles)
 
-<a name="4.1.4"></a>
-#### 4.1.4 Para-Virtualization method (Hardware support)
+<a name="3.0"></a>
+# 3. Scope
 
-This method basically is a mixture between the software only para-virtualization method and the direct assignment method (including the device sharing method) where the frontEnd driver which is running on the workload is a standard off the shelf driver and the backEnd driver is implemented straight in hardware logic (bypassing the hypervisor with hardware support from an IOMMU and SR-IOV) as shown in **Figure 7**.
+Within the framework of the Common Telecom cloud infrastructure vision, there are three levels of documents needed to describe components and allow the practical application of the systems. They are, as highlighted in **Figure 1**:  **Reference Model**, **Reference Architecture**, **Reference Implementation**, and **Reference Conformance**.
 
-Unlike the software only para-virtualized interfaces, this method provides better performance as it by-passes the hypervisor and unlike Direct Assignment methods, this method doesn’t require proprietary drivers to run in the workload and hence this method makes workloads portable.
+<p align="center"><img src="./figures/tech_scope.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 1:</b> Scope of CNTT</p>
 
-However, this method doesn’t provide the same level of flexibility as the software only para-virtualization method as migrating workloads from one host to another is more challenging due to the hardware presence and the state it holds for the workloads using it and therefore should also not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+<a name="3.1"></a>
+## 3.1 Specification Types
 
-<p align="center"><img src="./figures/tech_virtio_hw.png" alt="vitio_hw" title="Document Types" width="50%"/></p>
-<p align="center"><b>Figure 7:</b> Para-Virtualization method (with hardware support).</p>
+- **Reference Model (RM)**: focuses on the __**Infrastructure Abstraction**__ and how services and resources are exposed to VNFs/CNFs. It needs to be written at a high enough level that as new **Reference Architectures** and **Reference Implementations** are added, the model document should require few or no changes. Additionally, the Reference Model is intended to be neutral towards VMs or Containers.
+- **Reference Architecture (RA)**: High level Infrastructure system components and their interactions that takes the Reference Model goals and maps them to something that has components in the real world.  It is expected that at least one, but not more than a few, Reference Architecture will conform to the Reference Model.  The intention is, whenever possible, to use existing elements, rather than specify entirely new architectures in support of the high-level goals specified in the **Reference Model**.
```

Comment:
Reference Architectures should define all infrastructure components and properties which have effect on the VNF run time, deployment time or design time properties.

Reply:
What Gergely said! Plus:
A) Don't forget about the RA as they pertain to APIs
B) Small correction, below
> 
>"It is expected that at least one, but not more than a few, Reference Architecture will conform to the Reference Model."

Suggest: "It is expected that at least one, but not more than a few Reference Architectures will be created, and they will conform to the Reference Model."

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/ead6604b89361b525bdf266ed7698625472893d9